### PR TITLE
add : kayak

### PR DESCRIPTION
**Pull request by kiyaGu**
I need to merge my branch with the master branch.
what is this kayak

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+            "name": "Lhanna",
+            "age": 20
+        },
+        {
+            "name": "Gaj",
+            "age": -1
+        },
 {
 	       "name": "addis",
 	       "age": 20
```
Comment (ssozi91) [inline people.json]: lie

File: people.json [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 	     },
-	     {
-	       "name": "Gaj",
-	       "age": -1
-	     }]
-
+        {
+            "name": "Kiya",
+            "age": 20
```
Comment (ssozi91) [inline people.json]: lie lie

File: people.json [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+            "name": "Lhanna",
+            "age": 20
+        },
+        {
+            "name": "Gaj",
+            "age": -1
+        },
+        {
+            "name": "Kiya",
+            "age": 20
```
Comment (ssozi91) [inline people.json]: lie

**Comment by addiskeb12** [comment]
it looks good

**Pull request merged**